Bigtable/src/Connection/Grpc.php

```diff
@@ -0,0 +1,544 @@
+        return $this->send([$this->bigtableInstanceAdminClient, 'createAppProfile'], [
+            $parent,
+            $appProfileId,
+            $appProfile,
+            $this->addResourcePrefixHeader($args, $parent)
+        ]);
+    }
+
+    /**
+     * @param array $args
+     */
+    public function getAppProfile(array $args)
+    {
+        $name = $this->pluck('name', $args);
+        return $this->send([$this->bigtableInstanceAdminClient, 'getAppProfile'], [
+            $name,
+            $this->addResourcePrefixHeader($args, $name)
+        ]);
+    }
+
+    /**
+     * @param array $args
+     */
+    public function listAppProfiles(array $args)
+    {
+        $parent = $this->pluck('parent', $args);
+        return $this->send([$this->bigtableInstanceAdminClient, 'listAppProfiles'], [
+            $parent,
+            $this->addResourcePrefixHeader($args, $parent)
+        ]);
+    }
+
+    /**
+     * @param array $args
+     */
+    public function updateAppProfile(array $args)
+    {
+        $name = $this->pluck('name', $args);
+        $appProfile = $this->pluck('appProfile', $args);
+        $updateMask = $this->pluck('updateMask', $args);
+        return $this->send([$this->bigtableInstanceAdminClient, 'updateAppProfile'], [
+            $appProfile,
+            $updateMask,
+            $this->addResourcePrefixHeader($args, $name)
+        ]);
+    }
+
+    /**
+     * @param array $args
+     */
+    public function deleteAppProfile(array $args)
+    {
+        $name = $this->pluck('name', $args);
+        $ignoreWarnings = $this->pluck('ignoreWarnings', $args);
+        return $this->send([$this->bigtableInstanceAdminClient, 'deleteAppProfile'], [
+            $name,
+            $ignoreWarnings,
+            $this->addResourcePrefixHeader($args, $name)
+        ]);
+    }
+
+    /**
+     * @param array $args
+     */
+    public function getIamPolicy(array $args)
+    {
+        $resource = $this->pluck('resource', $args);
+        return $this->send([$this->bigtableInstanceAdminClient, 'getIamPolicy'], [
+            $resource,
+            $this->addResourcePrefixHeader($args, $resource)
+        ]);
+    }
+
+    /**
+     * @param array $args
+     */
+    public function setIamPolicy(array $args)
+    {
+        $resource = $this->pluck('resource', $args);
+        $policy = $this->pluck('policy', $args);
+        return $this->send([$this->bigtableInstanceAdminClient, 'setIamPolicy'], [
+            $resource,
+            $policy,
+            $this->addResourcePrefixHeader($args, $resource)
+        ]);
+    }
+
+    /**
+     * @param array $args
+     */
+    public function createTable(array $args)
+    {
+        $parent = $this->pluck('parent', $args);
+        $tableId = $this->pluck('tableId', $args);
+        return $this->send([$this->bigtableTableAdminClient, 'createTable'], [
+            $parent,
+            $tableId,
+            new Table(),
```

Comment:
Where is this Table class defined?

Reply:
This `Table` object is a protobuf object.  It should be passed in as a parameter, and should not be created here.

Reply:
Specifically, it's defined [here](https://github.com/googleapis/googleapis/blob/master/google/bigtable/admin/v2/table.proto#L33)

Reply:
Thanks.
Provided that this will be changed soon, my comment wouldn't be relevant anymore.
But, if it were to be used here, shouldn't we add a generated PHP class of Table in this PR too?
(It seems there is nothing checked in for BigTable yet).

Reply:
The generated proto classes live over here: https://github.com/googleapis/proto-client-php/blob/master/src/Google/Cloud/Bigtable/Admin/V2/Table.php

We are working on importing them into this repository currently.

Reply:
Thanks for explanation. It's clear to me now. :)